src/data/formDataObj.js

```diff
@@ -0,0 +1,40 @@
+const submitButton = document.getElementById("submit");
+submitButton.addEventListener("click", function (event) {
+    event.preventDefault();
+
+    function dataObj (candidate, interviewer, date, level, comments, evaluation) {
+        this.candidate = candidate;
+        this.interviewer = interviewer;
+        this.date = date;
+        this.level = level;
+        this.comments = comments;
+        this.evaluation = evaluation;
+    }
+
+    const candidate = document.getElementById("candidate");
+    const interviewer = document.getElementById("interviewer");
+    const date = document.getElementById("date");
+
+    const level = document.getElementsByClassName("radioButtons");
+
+    const levelArr = [];
+    level.forEach(function(el) {
```

Comment:
can you maybe use another method here to achieve this functionality, other than forEach? You want to create an array with the same length as the original one.

Reply:
Right. I'll use map then.

Reply:
there are several similar patterns, make sure to use map in all the areas :)